VMEncryption/main/handle.py

```diff
@@ -779,6 +863,38 @@ def enable_encryption():
                       code=str(CommonVariables.unknown_error),
                       message=message)
 
+def is_migration_allowed(disk_util, existing_passphrase_file):
```

Comment:
it would be good to have unit tests of the positive case, and the three negative cases

Reply:
Adding unit test for this will need refactoring done by Shiv for Single Pass. That is not available in dual pass. Hence, it might not be straight forward. I will discuss this further with you offline.